### PR TITLE
Fix/stay on single page

### PR DESCRIPTION
**Pull request by lusy**
What do you think?

### CODE REVIEW

File: forms/tests/test_form_response_status.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        self.client = Client()
+        self.client.force_login(self.admin_user)
+
+    def test_update_status_works(self):
+        payload = {
+            "status": "V",
+        }
+        self.client.post(make_url(self.response), data=payload)
+        updatedResponse = FormResponse.objects.get(id=self.response.id)
```
Comment (k-nut) [inline forms/tests/test_form_response_status.py]: I feel like pep8 would like us to call this `updated_response`. Other then that this looks good!

**Comment by k-nut** [comment]
One more thing. It probably makes sense to rebase this on `development` now and merge into that branch. We decided that we will gather features on that branch and then push it in one big blob to `master` once we are happy with the overall outcome.

File: forms/tests/test_form_response_status.py (hunk 1 above)
Comment (lusy) [reply forms/tests/test_form_response_status.py]: ah, that's very much possible, I keep forgetting this and mixing it up! on it.

**Comment by lusy** [comment]
let's do the rebase together tomorrow!

**Pull request merged**